org.mintguide.WebAppReader.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: /app/webappreader.sh
+finish-args:
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --share=network
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    QTWEBENGINE_DISABLE_SANDBOX: '1'
+modules:
+  - python3-requirements.json
```

Comment:
this is also missing.

Reply:
What you mean missing? Should I add all necessary files to this branch? And source code?

Reply:
How do you think flatpak-builder will locate that file `python3-requirements.json` ? out of thin air?